internal/internal_workflow_testsuite.go

```diff
@@ -402,6 +402,13 @@ func (env *testWorkflowEnvironmentImpl) newTestWorkflowEnvironmentForChild(param
 	childEnv.workflowInfo.CronSchedule = cronSchedule
 	childEnv.workflowInfo.ParentWorkflowNamespace = env.workflowInfo.Namespace
 	childEnv.workflowInfo.ParentWorkflowExecution = &env.workflowInfo.WorkflowExecution
+
+	searchAttrs, err := serializeSearchAttributes(params.SearchAttributes, params.TypedSearchAttributes)
+	if err != nil {
+		return nil, serviceerror.NewInvalidArgument("Invalid search attributes")
```

Comment:
I think we should pass the original error from `serializeSearchAttributes `, either as is or wrapped.